src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task4_2.java

```diff
@@ -0,0 +1,19 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+        import java.util.Scanner;
+
+public class Task4_2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int sum = 0;
+        System.out.println("Enter a last number please:");
+        int SecondNUmber = scanner.nextInt();
+        System.out.println("The sum for our numbers is:");
+        int i = 0;
+        do {
+            sum = sum + i;
+            i++;
+        } while (i >= 0 && i <= SecondNUmber);
```

Comment:
the same here, i >=0 is not necessary in condition

Reply:
Done